src/data/types/items.ts

```diff
@@ -13,22 +24,51 @@ export const ITEMS_SCHEMA = {
   electric_thermal_storage_and_slab: { type: 'string' },
   electric_vehicle_charger: { type: 'string' },
   electric_wiring: { type: 'string' },
+  energy_audit: { type: 'string' },
   evaporative_cooler: { type: 'string' },
+  floor_insulation: { type: 'string' },
   geothermal_heating_installation: { type: 'string' },
-  heat_pump_air_conditioner_heater: { type: 'string' },
+  heat_pump_air_conditioner_heater: { type: 'string' }, // TODO remove
   heat_pump_clothes_dryer: { type: 'string' },
   heat_pump_water_heater: { type: 'string' },
   new_electric_vehicle: { type: 'string' },
+  new_plugin_hybrid_vehicle: { type: 'string' },
   non_heat_pump_clothes_dryer: { type: 'string' },
   non_heat_pump_water_heater: { type: 'string' },
   other: { type: 'string' },
+  other_heat_pump: { type: 'string' },
+  other_insulation: { type: 'string' },
+  other_weatherization: { type: 'string' },
   rooftop_solar_installation: { type: 'string' },
   smart_thermostat: { type: 'string' },
   used_electric_vehicle: { type: 'string' },
-  weatherization: { type: 'string' },
+  used_plugin_hybrid_vehicle: { type: 'string' },
+  wall_insulation: { type: 'string' },
+  weatherization: { type: 'string' }, // TODO remove
   whole_house_fan: { type: 'string' },
+  window_replacement: { type: 'string' },
 } as const;
```

Comment:
I think we're missing `electric_lawn_equipment`?

Reply:
For the sake of completeness I'll make an argument that `water_efficiency` should be represented, but I get how that's not really our thing. Energy auditors are taught to consider water efficiency as energy efficiency because it a utility cost and it takes energy to clean water, build water infrastructure and transport water. I also think a climate impacted world, and the same audience who is thinking about electrification, will find it relevant. Do I think we should include it, though? I have no idea.

Reply:
We have `electric_outdoor_equipment`; slightly more general, but I feel like it's not worth making distinctions finer than that.

I get the argument for water efficiency, but I think that's a separate question from how to split up existing categories. It seems pretty clearly not a subset of any of the existing items.

Reply:
Ah, I missed `electric_outdoor_equipment`. LGTM!